app/scripts/modules/kubernetes/securityGroup/configure/wizard/upsert.controller.js

```diff
@@ -157,16 +157,19 @@ module.exports = angular.module('spinnaker.securityGroup.kubernetes.create.contr
     this.accountUpdated = function() {
       $q.all({
         accountDetails: accountService.getAccountDetails($scope.securityGroup.account),
-        loadBalancers: loadBalancerReader.listLoadBalancers('kubernetes'),
       }).then(function(backingData) {
         $scope.namespaces = backingData.accountDetails.namespaces;
-        $scope.loadBalancers = getLoadBalancerNames(backingData.loadBalancers);
         ctrl.namespaceUpdated();
       });
     };
 
     this.namespaceUpdated = function() {
       updateSecurityGroupNames();
+      $q.all({
```

Comment:
Ah, good point. I'll refactor the `accountUpdated` method as well since it's in a similar state.